Documentation/operations/troubleshooting.rst

```diff
@@ -290,24 +284,24 @@ command:
 
 .. code:: bash
 
-    $ hubble status --server localhost:4245
+   hubble status --server localhost:4245
 
 This command should return an output similar to the following:
 
-.. code:: bash
+.. code:: shell-session
 
-    Healthcheck (via localhost:4245): Ok
-    Max Flows: 16384
-    Current Flows: 16384 (100.00%)
+   Healthcheck (via localhost:4245): Ok
+   Max Flows: 16384
+   Current Flows: 16384 (100.00%)
 
 For convenience, you may set and export the ``HUBBLE_DEFAULT_SOCKET_PATH``
```

Comment:
```suggestion
For convenience, you may set and export the ``HUBBLE_SERVER``
```

The old environment variable (`HUBBLE_DEFAULT_SOCKET_PATH`) still works but is deprecated/not recommended anymore.